client/src/components/Header.js

```diff
@@ -39,6 +40,13 @@ export default class Header extends Component {
 
   componentDidMount() {
     this.loadBasicRepoData()
+    setInterval(() => {
```

Comment:
I think it's better to put this `setInterval` after `fetchLatestIndex()` is done, as `fetchLatestIndex()` would also involving getting the latest index.